UnleashClient/__init__.py

```diff
@@ -227,8 +226,14 @@ def is_enabled(self,
         :return: Feature flag result
         """
         context = context or {}
+
+        # Update context with static values
         context.update(self.unleash_static_context)
 
+        # Update context with optional values
+        if 'currentTime' not in context.keys():
+            context.update({'currentTime': datetime.now()})
```

Comment:
I might need to steal this to the Node.SDK. 

The only reason I kept this in the operator itself is that it does give an additional cost for every `isEnabled` call to create currentTime instance, even if it is mostly not needed (which will be 99% of the cases). On the other hand, datetime.now() probably has an insignificant cost?

Reply:
tldr after a few rabbit holes: Yes for modern CPUs.  

I was mainly putting it there since it seemed that it might be handy for other uses (on reflection though, not 100% sure what those could be).  I'd be OK with moving this into the `Constraint` check.  

Do you think you'd use `currentTime` in other features?

Reply:
not sure yet. I do not see a big issue moving it out to the context creation later if we find use for it. I would prefer us to have it in the constraint for now.

Reply:
I've moved this down into the `Constriant.apply()` function: https://github.com/Unleash/unleash-client-python/pull/192/files#diff-5a34048a72d9dbcc1cf64fe80a13119b0b40ee924c5cfc90a0352c30e7036ac4R162